packages/client/components/FacilitatorMenu.tsx

```diff
@@ -36,6 +45,10 @@ export default createFragmentContainer(FacilitatorMenu, {
   meeting: graphql`
     fragment FacilitatorMenu_meeting on NewMeeting {
       id
+      facilitatorUserId
+      meetingMembers {
+        userId
```

Comment:
-1 this will promote a potentially offline meeting member
let's filter by `user.isConnected`